apis/v1beta1/shared_types.go

```diff
@@ -58,6 +61,13 @@ type ParentReference struct {
 	// Gateway has the AllowedRoutes field, and ReferenceGrant provides a
 	// generic way to enable any other kind of cross-namespace reference.
 	//
+	// ParentRefs from a Route to a Service in the same namespace are "producer"
+	// routes, which apply default routing rules to inbound connections from
+	// any namespace to the Service. ParentRefs from a Route to a Service in a
+	// different namespace are "consumer" routes, and these routing rules are
+	// only applied to outbound connections to the Service from the same
+	// namespace as the Route.
```

Comment:
```suggestion
	// different namespace are "consumer" routes, and these routing rules are
	// only applied to outbound connections originating from the same namespace
	// as the Route, for which the intended destination of the connections are a
	// Service targeted as a ParentRef of the Route.
```

@keithmattix More verbose, but is this perhaps more explicitly clear?

Reply:
This update makes sense to me, maybe the section about consumer routes could justify a new paragraph to help split it up a bit more?

Reply:
Yeah I like this better, and I agree with Rob that at least a reference to some consumer route docs make sense